linebacker/src/main/scala/io/chrisdavenport/linebacker/Quarterback.scala

```diff
@@ -23,11 +22,8 @@ trait Quarterback[F[_], K] {
   def fleaFlicker[A](fa: F[A], initial: K, end: K)(implicit F: Async[F]): F[A] =
     for {
       iEC <- select(initial)
-      _ <- Async.shift(iEC)
-      aE <- fa.attempt
-      eEC <- select(end)
-      _ <- Async.shift(eEC)
-      a <- Applicative[F].pure(aE).rethrow
+      endEC <- select(end)
+      a <- Async[F].bracket(Async.shift[F](iEC))(_ => fa)(_ => Async.shift(endEC))
```

Comment:
so this is like the third time I see this. I think this is pointing to the need of either some package object or object definition of something like:

```scala
private[linebacker] final def dualShift[F[_], A](initialEc: ExecutionContext, endEc: ExecutionContext, action: F[A])(implicit F: Async[F]) = 
  F.bracket(Async.shift[F](initialEc))(_ => fa)(_ => Async.shift[F](encEc))
```

Then just calling `dualShift(ec1, ec2, fa)` in different modules and you save all the copypasta.